src/Phinx/Console/Command/Create.php

```diff
@@ -165,12 +165,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $path = realpath($path);
         $className = $input->getArgument('name');
+        if ($className === null) {
+            $className = "V" . Util::getCurrentTimestamp();
+            $fileName = Util::getCurrentTimestamp() . '_migration.php';
```

Comment:
You shouldnt call this twice, as this can lead to leap second issues.
Best to store in a variable and reuse that one.

Reply:
oh, yeah, duh

Reply:
done